From 854ace92eab725ef96ff98926809ce5829643968 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Fri, 4 Sep 2020 17:43:55 +0200
Subject: [PATCH 1/1] espresso7420: remove unused symbol CONFIG_DEFAULT_CONSOLE

Symbol CONFIG_DEFAULT_CONSOLE in include/configs/espresso7420.h is not used
for espresso7420_defconfig.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 include/configs/espresso7420.h | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/include/configs/espresso7420.h b/include/configs/espresso7420.h
index 5aeb009f03..2495db93f8 100644
--- a/include/configs/espresso7420.h
+++ b/include/configs/espresso7420.h
@@ -18,9 +18,6 @@
 #define CONFIG_SPL_STACK		CONFIG_IRAM_END
 #define CONFIG_SYS_INIT_SP_ADDR		CONFIG_IRAM_END
 
-/* select serial console configuration */
-#define CONFIG_DEFAULT_CONSOLE	"console=ttySAC1,115200n8\0"
-
 /* DRAM Memory Banks */
 #define SDRAM_BANK_SIZE		(256UL << 20UL)	/* 256 MB */
 
-- 
2.28.0

